plugins/inputs/webhooks/webhooks.go

```diff
@@ -105,11 +103,27 @@ func (wb *Webhooks) AvailableWebhooks() []Webhook {
 }
 
 func (wb *Webhooks) Start(acc telegraf.Accumulator) error {
-	go wb.Listen(acc)
+	r := mux.NewRouter()
+
+	for _, webhook := range wb.AvailableWebhooks() {
+		webhook.Register(r, acc)
+	}
+
+	wb.srv = &http.Server{Handler: r}
+
+	ln, err := net.Listen("tcp", fmt.Sprintf("%s", wb.ServiceAddress))
+	if err != nil {
+		acc.AddError(fmt.Errorf("E! Error starting server: %v", err))
```

Comment:
I think we should return this error, making it fatal and preventing Telegraf from starting.  This probably indicates the port is in use or a privilege issue, and won't resolve, so I think exit is the right action even though it is technically a change in behavior.